src/Views/HomeView/HomeView.js

```diff
@@ -1,6 +1,29 @@
 import React from "react";
+import groupImage from "./../../assets/DiverseGroup.png";
+import InformationBlock from "../../Components/InformationBlock/InformationBlock";
+import PreviewBlocks from "../../Components/PreviewBlocks/PreviewBlocks";
+
+import "./HomeView.scss";
+
 const HomeView = () => {
-  return <h1>hi</h1>;
+  return (
+    <div className="home-container">
+      <div className="welcome-container">
+        <img
+          src={groupImage}
+          alt="image-of-diverse-people"
```

Comment:
alt tag could be either "diverse people" or "illustration of diverse people", no need for hyphens since alt tags are for screen readers. 

(using an image in the alt tag is breaking the CI build, so just wanted to call out alternative language)

Reply:
Also my PR i submitted a few minutes ago, includes the updated image if you want to add that in with these changes if my PR gets merged first :) 

Looks sooo good!!